docs/source/GETTING_STARTED.rst

```diff
@@ -22,11 +22,11 @@ Google Cloud Setup
 
 .. note:: The recent success of deep learning has been critically dependent on accelerated hardware like GPUs. Similarly, the strength of the DeepCell Kiosk is its ability to recruit and scale GPU nodes based on demand. Google does not include these GPU nodes by default as part of its free tier thus necessitating the upgrade. For more information, please refer to `Google's blog post on the subject <https://cloud.google.com/blog/products/gcp/gpus-service-kubernetes-engine-are-now-generally-available>`_
 
-4. You will also need to `apply <https://cloud.google.com/compute/quotas>`_ for a quota of at least 1 GPU using the ``GPUs (all regions)`` option. Please also request at least 16 *In-use IP addresses* for the *Compute Engine API* of your region (by default ``us-west1``). This may take some time, as Google will need to approve each these requests.
+4. Apply for a `quota of at least 1 GPU <https://cloud.google.com/compute/quotas#gpus>`_ and at least `16 *In-use IP addresses* <https://cloud.google.com/compute/quotas#ip_addresses>`_ for the *Compute Engine API* of your region (by default ``us-west1``). This may take some time, as Google will need to approve each these requests.
 
 .. note:: Google offers a number of GPU types. The DeepCell Kiosk uses `nvidia-tesla-t4` GPUs for inference by default.
 
-5. Create a `cloud storage bucket <https://cloud.google.com/storage/docs/creating-buckets>`_. This will be used to store data and models. Record the bucket name (as with project ID, you will need this during Kiosk configuration). Please do not use underscores (`_`) in your bucket name. Your bucket should follow the organizational structure that follows:
+5. Create a Standard class `cloud storage bucket <https://cloud.google.com/storage/docs/creating-buckets>`_ with fine-grained access control in the default region of your project. This will be used to store data and models. Record the bucket name (as with project ID, you will need this during Kiosk configuration). Please do not use underscores (`_`) in your bucket name. Your bucket should follow the organizational structure that follows:
 
```

Comment:
I understand what Noah is saying but the opening sentence is overly complicated. 

Maybe: Create a `cloud storage bucket in the default region of your project (this should be a "Standard class" bucket, which you can select using fine-grained access control).